### PR TITLE
Cart props - cart dispatch funcs in Main

### PR DESCRIPTION
**Pull request by rrodrig003**

### CODE REVIEW

File: client/actions/cartActions.js [hunk 1]
```diff
@@ -6,15 +6,18 @@ export const getCart = cartItems => ({ type: GET_CART, cartItems });
 // eslint-disable-next-line arrow-parens
 export const fetchCartBySession = id => dispatch => {
```
Comment (KevinY90) [inline client/actions/cartActions.js:7]: I don't think fetchCartBySession and fetchCartByUser is needed if they're going to be called from the main component, let me know what you think!

File: client/components/Cart.jsx [hunk 2]
```diff
@@ -8,42 +8,40 @@ const propTypes = {
     productId: PropTypes.number.isRequired,
   })).isRequired,
   products: PropTypes.arrayOf(PropTypes.shape({
-    productId: PropTypes.number.isRequired,
-    name: PropTypes.number.isRequired,
-    price: PropTypes.number.isRequired,
+    id: PropTypes.number.isRequired,
+    name: PropTypes.string.isRequired,
+    price: PropTypes.string.isRequired,
```
Comment (KevinY90) [inline client/components/Cart.jsx:13]: price should be a number

File: client/components/Main.jsx [hunk 3]
```diff
@@ -6,21 +6,33 @@ import Navbar from '../containers/NavbarContainer';
...
 };
 
 export default class Main extends Component {
   componentDidMount() {
-    const { getProducts, onLoad } = this.props;
-    onLoad();
-    getProducts();
+    const { getProducts, onLoad, getCartByUser, getCartBySession, signedIn, userId, sessionId } = this.props;
+
+    if (onLoad() && getProducts()) {
```
Comment (KevinY90) [inline client/components/Main.jsx:29]: I'm not sure if we need this, I think we could take advantage of either middleware or existing action creators to get a session/user's order

**Review by KevinY90 — CHANGES_REQUESTED** [review]

File: client/components/Cart.jsx (hunk 2 above)
Comment (rrodrig003) [reply client/components/Cart.jsx:13]: Yes, price should be a number but its cominh through as a string, thats why I changed it

**Pull request closed without merging**